### ISSUE TITLE
Is it possible to avoid interpretation of institutionCode when the basis OfRecord is MaterialSample?

### ISSUE THREAD

**Issue by EstebanMH-SiB** [gbif/portal-feedback#3785]
Hi,

We have a [dataset](https://www.gbif.org/es/dataset/b9461088-6f97-49b0-8f76-17e5be9e2df0) with mixed basisOfRecord, HumanObservation and MaterialSample.

The problem is that we get the issue "institution match none" for the [records with MaterialSample](https://www.gbif.org/occurrence/search?dataset_key=b9461088-6f97-49b0-8f76-17e5be9e2df0&issue=INSTITUTION_MATCH_NONE), I guess is because it try to match it with GRSciColl, but this is not part of a collection, just indirect records (footprints, hair, marks, etc).
And it looks estrange that we get the issue just for 24 records when the whole dataset has the same institutionCode.

So, we should like to know if there is a way to avoid that interpretation, maybe just use the interpretation when collectionCode is docummented.

Thanks in advance and have a good day,

**Comment by ManonGros** [comment]
Good question, I don't know if this is something we could do.
The only two possible solutions I can think of on the side of the publisher are:

- leave the institution code and ID fields empty
- or create an entry for the on the GRSciColl side

I don't think there would be an way to do this on GBIF's side. @marcos-lg any thoughts on this?

**Comment by albenson-usgs** [comment]
As I've raised elsewhere I think GBIF's interpretation that `institutionCode` and `institutionID` must always = a GRSciColl entry is flawed. MaterialSamples will not always be housed in collections. For instance eDNA datasets which are MaterialSamples may not be managed by a museum in a collection. It could be managed by an observation or monitoring network instead. However there is still an institution that has "custody of the object(s) or information referred to in the record." Maybe the flags of not matching to GRSciColl should only happen for when basisOfRecord is PreservedSpecimen, LivingSpecimen, and FossilSpecimen?

**Comment by timrobertson100** [comment]
> As I've raised elsewhere I think GBIF's interpretation that institutionCode and institutionID must always = a GRSciColl entry is flawed

Is the problem here perhaps the way it is displayed as an "issue"? There will presumably be Material that will be housed or connected to an institution so it's reasonable to link where possible, but perhaps it simply shouldn't be flagged as an issue that no link can be found?

**Comment by albenson-usgs** [comment]
Yes I think so although this is not my Github issue so I may have overstepped my bounds to comment. When something is labeled as an issue it looks like it needs to be corrected but in this case there is nothing to correct as the flag doesn't make sense for these data. Seems like we need something closer to a "note" than a flag. A note indicating "we weren't able to make a match to GRSciColl here" and the data manager can review that and determine that that is indeed the way it should be so there is no action to take. Also that downstream users understand that an inability to match does not indicate that something is incorrect.

**Comment by EstebanMH-SiB** [comment]
We don't really like neither options of @ManonGros, bacause both are incorrect from the publisher perspective. This organization doesn't have a collection and never will and they don't want to have their records without institutionCode, because is important for visibility.

We totally agree with @albenson-usgs in both of her comments. A "note" should be ideal for this kind of problem, maybe in grey like in the Data Validator and with a different message.
Is it possible to keep a flag when basisOfRecord is PreservedSpecimen, LivingSpecime or FossilSpecimen and a note in other cases?

**Comment by ManonGros** [comment]
Thank you all for your input.
What we will do is: 

+ If materialSample records have an institution and collection code or ID that link to GRSciColl, we will link the record to GRSciColl. If there is no match in GRSciColl, silently ignore it (no flag).
+ Have the GRSciColl-related flags in grey (info)

@marcos-lg correct me if needed
